lib/instrumentation/pg.js

```diff
@@ -89,11 +91,23 @@ module.exports = function initialize(agent, pgsql, moduleName, shim) {
     })
   }
 
+  const pgVersion = shim.require('./package.json').version
   // wrapping for JS
   shim.recordQuery(
     pgsql && pgsql.Client && pgsql.Client.prototype,
     'query',
     function wrapJSClientQuery(shim) {
+      if (semver.satisfies(pgVersion, '>=7.0.0') &&
+      typeof arguments['3'][0] === 'string') {
```

Comment:
this feels a little odd that you're mixing object key access and indices.  You could either do `arguments[3][0]` or something a little clearer like

```js
   const [ shim, , , queryArgs = []] = args
   if (semver.satisfies(pgVersion, '>=7.0.0') &&
     typeof queryArgs[0] === 'string') {
```

Also if for some reason someone did `client.query()` this would blow up right?

Reply:
might also be good to put a comment explaning this semver and args check for future team

Reply:
`client.query()` called with no args causes a `pg` level error: `Client was passed a null or undefined query`